rules/sinks/storages/facebooklocalcache/java.yaml

```diff
@@ -0,0 +1,13 @@
+sinks:
+  - id: Storages.Facebook.LocalCache
+    name: Facebook Android Local Cache
+    patterns:
+      - "(?i)(.*)(com.facebook.Session.saveTokenToCache(.*)"
+    tags:
+      law: GDPR
+  - id: Storages.Facebook.SaveCookie
+    name: Facebook Android Cookie
+    patterns:
+      - "(?i)(.*)(com.facebook.AuthorizationClient$WebViewAuthHandler.saveCookieToken:void(java.lang.String))(.*)"
```

Comment:
Remove .* from beginning

Reply:
Similar to last comments we can update this pattern to `com.facebook.AuthorizationClient$WebViewAuthHandler.saveCookieToken.*` to increase coverage